### PR TITLE
Fixes code style module

### PR DESCRIPTION
**Pull request by AndriyPt**

### CODE REVIEW

File: ansible/roles/ci/code_style_check/cpp/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- name: Read all packages paths in project
+  shell: find -type f -name package.xml -printf "%h\n" chdir={{project_sources_path}}
+  register: package_paths_list
+
+- name: Set variable to workaround ansible type evaluation issue
+  set_fact:
+    package_paths_list_stdout_lines: "{{package_paths_list.stdout_lines}}"
+
+- name: Find C++ files in each package
+  shell: find -type f \( -name "*.h" -o -name "*.cpp" \) -printf "%p "
```
Comment (ugocupcic) [inline ansible/roles/ci/code_style_check/cpp/tasks/main.yml]: I'm using `*.hpp` a lot

File: ansible/roles/ci/code_style_check/python/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+# Playbook for Python code style check
+
+- name: check if project_sources_path variable was set
+  fail: msg="Variable project_sources_path was not set"
+  when: project_sources_path is not defined
+
+- name: Read all packages paths in project
```
Comment (ugocupcic) [inline ansible/roles/ci/code_style_check/python/tasks/main.yml]: some duplication with cpp style checks. Probably ok though

File: ansible/roles/ci/unit_tests/tasks/main.yml [hunk 3]
```diff
@@ -1,6 +1,32 @@
...
+
+- name: Read all packages names in workspace
+  shell: find -type f -name package.xml -printf "%h\n" | xargs basename -a chdir={{ros_workspace}}
+  register: workspace_packages_list
+
+- name: Read all packages names in project
+  shell: find -type f -name package.xml -printf "%h\n" | xargs basename -a chdir={{project_sources_path}}
+  register: project_packages_list
+
+- name: Set blacklisted packages using lists substraction
```
Comment (ugocupcic) [inline ansible/roles/ci/unit_tests/tasks/main.yml]: Use a more meaningful name? **Blacklist non-project packages** or something like this

File: ansible/roles/ci/code_style_check/cpp/tasks/main.yml (hunk 1 above)
Comment (AndriyPt) [inline ansible/roles/ci/code_style_check/cpp/tasks/main.yml]: Fixed

File: ansible/roles/ci/code_style_check/python/tasks/main.yml (hunk 2 above)
Comment (AndriyPt) [inline ansible/roles/ci/code_style_check/python/tasks/main.yml]: There is small amount of duplication in the code. Lets leave it for now in order to get something working and I'll do refactoring after we get to some substantial functionality.

**Pull request merged**